skimage/transform/tests/test_thin_plate_splines_transform.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+import skimage as ski
+from skimage.transform._thin_plate_splines import (TPSTransform, _ensure_2d,
+                                                   tps_warp)
+
+SRC = np.array([
+    [0, 0],
+    [0, 5],
+    [5, 5],
+    [5, 0]
+])
+
+DST = np.array([
+    [5, 0],
+    [0, 0],
+    [0, 5],
+    [5, 5]
+])
+
+
+def test_zero_image_size():
+    with pytest.raises(ValueError):
+        tps_warp(np.zeros(0), TPSTransform())
+    with pytest.raises(ValueError):
+        tps_warp(np.zeros((0, 10)), TPSTransform())
+    with pytest.raises(ValueError):
+        tps_warp(np.zeros((10, 0)), TPSTransform())
```

Comment:
I think this test should be easy to [parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions). Let me know if you need a few more pointers on how to do that. :)